tests/test_alerts.py

```diff
@@ -281,6 +281,45 @@ def test_expired_alerts(self):
         data = json.loads(response.data.decode('utf-8'))
         self.assertEqual(data['alert']['status'], 'open')
 
+    def test_reopen_alerts(self):
+
+        # severity == warning -> status=open
+        response = self.client.post('/alert', data=json.dumps(self.warn_alert), headers=self.headers)
+        self.assertEqual(response.status_code, 201)
+        data = json.loads(response.data.decode('utf-8'))
+        self.assertEqual(data['alert']['status'], 'open')
+
+        alert_id = data['id']
+
+        # severity == normal -> status=closed
+        response = self.client.post('/alert', data=json.dumps(self.normal_alert), headers=self.headers)
+        self.assertEqual(response.status_code, 201)
+        data = json.loads(response.data.decode('utf-8'))
+        self.assertEqual(data['alert']['status'], 'closed')
+
+        # severity == warning -> status=open
+        response = self.client.post('/alert', data=json.dumps(self.warn_alert), headers=self.headers)
+        self.assertEqual(response.status_code, 201)
+        data = json.loads(response.data.decode('utf-8'))
+        self.assertEqual(data['alert']['status'], 'open')
+        self.assertEqual(data['alert']['duplicateCount'], 0)
+
+        # operator=closed -> status=closed
+        response = self.client.put('/alert/' + alert_id + '/status', data=json.dumps({'status': 'closed', 'text': 'operator action'}), headers=self.headers)
+        self.assertEqual(response.status_code, 200)
+        response = self.client.get('/alert/' + alert_id)
+        self.assertEqual(response.status_code, 200)
+        data = json.loads(response.data.decode('utf-8'))
+        self.assertEqual(data['alert']['status'], 'closed')
+
+        # severity == warning -> status=open
+        response = self.client.post('/alert', data=json.dumps(self.warn_alert), headers=self.headers)
+        self.assertEqual(response.status_code, 201)
+        data = json.loads(response.data.decode('utf-8'))
+        self.assertEqual(data['alert']['status'], 'open')
+        self.assertEqual(data['alert']['severity'], self.app.config['DEFAULT_NORMAL_SEVERITY'])
+        self.assertEqual(data['alert']['duplicateCount'], 0)
```

Comment:
currently severity is "warning" and duplicate count is "1" when it should be "normal" and "0".